bvt/op-ipmi-basic-bvt.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sorry one more comment.  The fact this (and the file below) is just called ipmi-basic-bvt I think is a little misleading.  It does an actual firmware update of the BMC and PNOR so I think that should be reflected in the file names, or at least in the comments for the files.